Sources/StreamChatUI/ChatMessageList/Attachments/ChatMessageGalleryView.swift

```diff
@@ -68,9 +68,11 @@ open class _ChatMessageGalleryView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
     
     /// Left container for previews.
     public private(set) lazy var leftPreviewsContainerView = ContainerStackView()
+        .withoutAutoresizingMaskConstraints
```

Comment:
I think this is a problem with our stack view, it should set this flag automatically when you add an arranged subview. Just like `UIStackView` does it.

Reply:
The stack view does it. I added it just for consistency of the code